R/Cohorts.R

```diff
@@ -14,18 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 #' Construct cohorts
 #'
 #' @description
 #' Creates a set of predefined cohorts in a cohort table. WARNING: this will delete all existing
 #' cohorts in the table!
 #'
 #' @param connectionDetails      The connection details to connect to the (Eunomia) database.
-#' @param cdmDatabaseSchema      The name of the database schema holding the CDM data.
-#' @param cohortDatabaseSchema   The name of the database schema where the cohorts will be written.
-#' @param cohortTable            The name of the table in the cohortDatabaseSchema where the cohorts
-#'                               will be written.
+#' @param cdmDatabaseSchema      Deprecated. The cdm must be created in the main schema.
+#' @param cohortDatabaseSchema   Deprecated. The cohort table will be created in the main schema.
+#' @param cohortTable            Deprecated. Cohort table will be named "main".
```

Comment:
Typo? I think the cohort table will be named 'cohort'?

Reply:
yep that's a typo.